stores/apps/selectors.js

```diff
@@ -0,0 +1,4 @@
+// this should probs be renamed so it's not apps.apps
+export const selectApps = (state) => state.apps.apps.apps
+export const selectNewsletterAPI = (state) => state.apps.newsletterApi
```

Comment:
There is no such `newsletterApi`. It's the same server as `apiServer`

Reply:
oh whoops!